spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/core/SpannerTransactionManagerTests.java

```diff
@@ -183,13 +191,13 @@ protected Tx getCurrentTX() {
 
 	@Test
 	public void testDoCommitNotStarted() {
-		TransactionManager transactionManager = Mockito.mock(TransactionManager.class);
+		TransactionManager transactionManager = mock(TransactionManager.class);
 
-		SpannerTransactionManager.Tx tx = Mockito.mock(SpannerTransactionManager.Tx.class);
+		SpannerTransactionManager.Tx tx = mock(SpannerTransactionManager.Tx.class);
 		ReflectionTestUtils.setField(tx, "transactionManager", transactionManager);
 
-		DefaultTransactionStatus status = Mockito.mock(DefaultTransactionStatus.class);
-		Mockito.when(status.getTransaction()).thenReturn(tx);
+		DefaultTransactionStatus status = mock(DefaultTransactionStatus.class);
+		when(status.getTransaction()).thenReturn(tx);
```

Comment:
Should this test verify that `.close()` did not get called?

Reply:
Done, PTAL @elefeint